quota-management/inventory/hosts

```diff
@@ -0,0 +1,5 @@
+[masters]
+master-[0:2].c1-osp.myorg.com
+
+[seed-hosts:children]
+masters
```

Comment:
This might be my lack of Ansible knowledge but shouldn't this be
```
[seed-hosts]
localhost ansible_connection=local
```

Reply:
so in this instances we are taking advantage of the fact that we are cluster operators, and using the `cluster-admin` user on the masters insteado of our own personal users. This is a strategy we use when we want to be able to move straight from an openshift-ansible run to an openshift-applier run.

Reply:
Looks great for a Admin section. Would be nice if defaultRequest: for type Container could be added to the limits section object. A lot of templates in -n openshift will not work without that.